akka-actor/src/main/scala/akka/routing/Resizer.scala

```diff
@@ -71,6 +94,7 @@ case object DefaultResizer {
       backoffRate = resizerConfig.getDouble("backoff-rate"),
       messagesPerResize = resizerConfig.getInt("messages-per-resize"))
 
+  @deprecated("User Resizer.fromConfig instead", since = "2.4.1")
```

Comment:
I need to think why/if this actually needs deprecation - is it not possible to make the "new" Resizer fit here?